### ISSUE TITLE
xterm overlap its window decoration

### ISSUE THREAD

**Issue by ghifarit53** [JLErvin/berry#89]
First of all, thanks for the amazing and fully customizable wm, i love the modular nature of it

Second, unfortunately i have an issue with xterm. Its window overlap its decoration, as you can see below
![2020-07-10_07 37 33](https://user-images.githubusercontent.com/59758342/87103870-6ee00900-c280-11ea-8eac-f0b25e34c919.png)

And here's a screenshot of urxvt, chrome, and thunar, all of it looks normal
![2020-07-10_07 37 28](https://user-images.githubusercontent.com/59758342/87103930-a058d480-c280-11ea-955e-173809184b6b.png)
![2020-07-10_07 40 53](https://user-images.githubusercontent.com/59758342/87103973-c0889380-c280-11ea-993f-1d3eb69c6724.png)
![2020-07-10_07 41 31](https://user-images.githubusercontent.com/59758342/87103981-c8483800-c280-11ea-94a1-dccb1f2b69d5.png)

Here's my berry config
```
#!/bin/bash

sxhkd -c ~/.config/berry/sxhkdrc &

xrdb -merge ~/.Xresources &

~/.fehbg &

~/.config/polybar/run.berry &

# Set decoration geometry
berryc border_width       0
berryc inner_border_width 0
berryc title_height       30
berryc top_gap            0

# Set decoration colors
berryc inner_focus_color         4f5666
berryc inner_unfocus_color       4f5666
berryc text_focus_color          4f5666
berryc text_unfocus_color        4f5666

# Other options
berryc smart_place "false"
berryc draw_text   "false"
berryc edge_lock   "false"
berryc set_font    "SF Mono-10"
```

and my .Xresources 
```
#include ".themes/onedark"

XTerm*geometry: 60x21
XTerm*faceName: SF Mono
XTerm*faceSize: 10

! font settings
URxvt*font: xft:SF Mono:size=10, xft
URxvt*letterSpace: -1

! appearance
URxvt*scrollBar: false
URxvt*scrollBar_right: false
URxvt*internalBorder: 12
URxvt*cursorBlink: true
URxvt*geometry: 60x18

! copy and pasting
URxvt.keysym.Shift-Control-V: eval:paste_clipboard
URxvt.keysym.Shift-Control-C: eval:selection_to_clipboard
URxvt.keysym.Control-Meta-c: builtin-string:
URxvt.keysym.Control-Meta-v: builtin-string:

URxvt.iso14755: false

! Xft settings
Xft.dpi: 96
Xft.antialias: true
Xft.hinting: true
Xft.rgba: none
Xft.autohint: true
Xft.hintstyle: hintfull
Xft.lcdfilter: lcdfilter

```
I've tried to resize the `XTerm*geometry` option, also using berryc window resizing, but still no change

I don't have any experience in C programming, especially XLib, but i really love this WM, so if you need more infos from me, i'll happily provide them

**Comment by ghifarit53** [comment]
sorry my bad, i don't think that this is a problem with berry, but xterm itself

**Comment by JLErvin** [comment]
hm, I was able to recreate this on my end. Let me see if I can come up with a fix - `berry` related or not.

**Comment by JLErvin** [comment]
Ok, I added a fix on the development branch, `hucklberry`. You can clone/use this branch for a temporary fix. I'll update and close this issues once it's merged into master.

[development branch here](https://github.com/JLErvin/berry/tree/huckleberry)

**Comment by ghifarit53** [comment]
It works! thank you so much

![2020-07-11_09 05 05](https://user-images.githubusercontent.com/59758342/87214338-b1274a00-c355-11ea-9afe-699515de82f6.png)

**Comment by ghifarit53** [comment]
Sorry i accidentally pressed close

**Comment by JLErvin** [comment]
Released as of version 0.1.6. Closing.
